examples/train.py

```diff
@@ -182,12 +181,12 @@ def setup_exps_rllib(flow_params,
         submodule = getattr(module, flags.exp_config)
     elif hasattr(module_ma, flags.exp_config):
         submodule = getattr(module_ma, flags.exp_config)
-        assert flags.rl_trainer == "RLlib", \
+        assert flags.rl_trainer.lower() == "RLlib".lower(), \
             "Currently, multiagent experiments are only supported through "\
             "RLlib. Try running this experiment using RLlib: 'python train.py EXP_CONFIG'"
     else:
         assert False, "Unable to find experiment config!"
-    if flags.rl_trainer == "RLlib":
+    if flags.rl_trainer.lower() == "rllib":
```

Comment:
why rename it? just for aesthetics?

Reply:
They can just type whatever they want and capitalization errors wont throw an error